public/scripts/init.js

```diff
@@ -57,10 +57,27 @@ window.addEventListener("load", async () => {
   const realtime = window.ls.container.get('realtime');
   const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
   let current = {};
-  window.ls.container.get('console').subscribe('project', event => {
-    for (let project in event.payload) {
-      current[project] = event.payload[project] ?? 0;
+  window.ls.container.get('console').subscribe(['project', 'console'], response => {
+    switch (response.event) {
+      case 'stats.connections':
+        for (let project in response.payload) {
+          current[project] = response.payload[project] ?? 0;
+        }
+        break;
+      case 'database.attributes.create':
+      case 'database.attributes.update':
+      case 'database.attributes.delete':
+        document.dispatchEvent(new CustomEvent('database.createAttribute'));
```

Comment:
This is a little confusing to me - why do we send `database.createAttribute` for case `database.attributes.delete`?

Reply:
This triggers a refetch in the UI 👍🏻